plugwise/helper.py

```diff
@@ -180,12 +191,12 @@ class SmileComm:
 
     def __init__(
         self,
-        host,
-        password,
-        username,
-        port,
-        timeout,
-        websession,
+        host: str,
+        password: str,
+        username: str,
+        port: str,
+        timeout: str,
```

Comment:
```suggestion
        timeout: float,
```

Reply:
Nope, its a float, you can do `2.5` seconds for example

Reply:
Sure, I'm flexible.